main.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"strings"
+
+	"github.com/fatih/color"
+	"github.com/rodaine/table"
+)
+
+func main() {
+	var repoURL string
+	var directory string
+	var skipFetch bool
+	var baseBranch string
+	var branchesString string
+	var err error
+	var diff []DiffCommitBranch
+
+	flag.StringVar(&repoURL, "repoUrl", "", "git repo Url. ssh and https supported")
+	flag.StringVar(&directory, "directory", "", "Location to store or retrieve from the repo")
+	flag.BoolVar(&skipFetch, "skipFetch", false, "Skip fetch. When set you may not be up to date with remote")
+	flag.StringVar(&baseBranch, "baseBranch", "master", "Base branch for the comparison.")
+	flag.StringVar(&branchesString, "branches", "", "Comma separated list of branches")
+	flag.Parse()
+
+	repo, err := loadOrClone(repoURL, directory, "origin", skipFetch)
```

Comment:
someone might not name this origin...

Reply:
👍 Yeap, currently it works only for origin. Ideally the remote should be configurable, and even give the ability for no remote so that you can check your local branches